site/docs/master/code-standards.md

```diff
@@ -43,7 +43,8 @@ Example:
     	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
     	corev1client "k8s.io/client-go/kubernetes/typed/core/v1"
     	corev1listers "k8s.io/client-go/listers/core/v1"
-        
+       
+        veleroapiv1 "github.com/vmware-tanzu/velero/api/v1" // for the new kubebuilder apis 
         velerov1api ""github.com/vmware-tanzu/velero/pkg/apis/velero/v1"
```

Comment:
`velerov1api` is a reused import alias.
Also looks like line#48 has a compiler error: unmatched `"`.

Reply:
`github.com/vmware-tanzu/velero/api/v1` no longer exists so this change is being reverted.